app/views/components/docs/option-select.yml

```diff
@@ -248,3 +248,26 @@ examples:
       - value: trinidad
         label: Trinidad & Tobago
         id: trinidad
+  with_option_lookup:
+    description: |
+      This example shows option_lookup used to map individual values to a set of values. This is used to avoid excess
+      data being sent to the client and to avoid the need for any parsing / deserialization of form-provided data.
+    data:
+      key: groups_of_clothes
+      title: Groups of clothes
+      options_container_id: groups_of_clothes
+      options:
+      - value: hats
+        label: Hats
+        id: hats
+      - value: socks
+        label: socks
+        id: socks
+      option_lookup:
```

Comment:
What does `option_lookup` actually do in the component? I can't see any changes to the component itself other than to this yml file, and I checked out the branch and removed the `option_lookup` data from this file, and everything rendered exactly as it had before. Have all the changes been committed?

Reply:
`option_lookup` maps a bunch of keys to the actual values they stand-in for. In `allowed_values` we use the _keys_ defined in `option_lookup` as the values which are then populated in each of the checkboxes. 

To go with an example, `closed_consultations` in for this policy finder maps to an array `['closed_consultation', 'consultation_outcome']`. In the component, the checkbox rendered with have the value `closed_consultation`, and when that checkbox is selected, it'll get mapped behind the scenes to the actual values which that key represents, so the search will be performed against doc types of `closed_consultation` and `consultation_outcome`.

Reply:
This documentation is for the component, and this isn't a change to the component, it's a change to the data that you're passing to the component. It's misleading to have this documentation because it suggests that the component accepts something called `option_lookup` when in fact it doesn't.

I agree that documenting this somewhere is a good idea, but this isn't the place for it.

Reply:
This has now been removed.